source/dub/commandline.d

```diff
@@ -383,16 +650,90 @@ class CommandArgs {
 		m_args = null;
 	}
 
-	/** Returns the list of unprocessed arguments and calls `dropAllArgs`.
+	/** Returns the list of unprocessed arguments, ignoring the app arguments,
+	and calls `dropAllArgs`.
 	*/
 	string[] extractRemainingArgs()
 	{
 		auto ret = m_args[1 .. $];
 		m_args = null;
 		return ret;
 	}
+
+	/** Returns the list of unprocessed arguments, including the app arguments
+		and calls `dropAllArgs`.
+
+	Returns:
+		Returns the list of unprocessed arguments with the app arguments
```

Comment:
this "Returns" section is a bit useless with the documentation before it saying the same already.